test/Microsoft.AspNet.Mvc.ModelBinding.Test/Metadata/DataAnnotationsMetadataDetailsProviderTest.cs

```diff
@@ -0,0 +1,149 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.DataAnnotations;
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc.ModelBinding.Metadata
+{
+    public class DataAnnotationsMetadataDetailsProviderTest
```

Comment:
seems to be missing tests similar to previous `DisplayFormatString_AttributesHaveExpectedPrecedence` and `TemplateHint_AttributesHaveExpectedPrecedence`